appendix-basic65.tex

```diff
@@ -7237,49 +7401,61 @@ \subsection{PLAY}
+Directives consist of a letter, followed by a digit. Directives apply to all
+future notes, until the parameter is changed by another directive.
 
 \begin{center}
 {\setlength{\tabcolsep}{1mm}
 \begin{tabular}{*{1}{|C{9mm}}|l|l|}
 \hline
-{\bf Char}  & {\bf Directive} & {\bf Argument Range} \\
+{\bf Character}  & {\bf Directive} & {\bf Argument Range} \\
 \hline
-\screentext{O} & Octave              & 0 - 6 \\
-\screentext{T} & Instrument Envelope & 0 - 9 \\
-\screentext{U} & Volume              & 0 - 9 \\
-\screentext{X} & Filter              & 0 - 1 \\
-\screentext{M} & Modulation          & 0 - 9 \\
-\screentext{P} & Portamento          & 0 - 9 \\
+\screentext{O} & Octave              & 0 -- 6 \\
+\screentext{T} & Instrument Envelope & 0 -- 9 \\
+\screentext{U} & Volume              & 0 -- 9 \\
+\screentext{X} & Filter              & 0 -- 1 \\
+\screentext{M} & Modulation          & 0 -- 9 \\
+\screentext{P} & Portamento          & 0 -- 9 \\
 \screentext{L} & Loop                & N/A   \\
 \hline
 \end{tabular}
 }
 \end{center}
 
-  The modulation directive will modulate your note by the magnitude you specify (1-9), or use 0 to not use it.
+  An octave is a range of notes from C to B. The default octave is 4,
+  representing the ``middle'' octave.
 
-    Similarly, the portamento directive will gently slide between consecutive notes at the speed you specify (1-9), or use 0 to not use it. Note that the gate-off behaviour of notes is disabled while portamento is enabled, and to re-enable it, you must disable portamento (P0).
+  Instrument envelopes describe the nature of the sound. See {\bf ENVELOPE}
+  for a list of default envelope styles, and information on how to adjust the
+  ten envelopes.
 
-    Add an {\bf L} directive (no argument needed) at the end of your string if you would like it to loop back to the
-  beginning of your string upon completion.
+  The modulation directive adds a pitch-based vibrato your note by the magnitude you
+  specify (1 -- 9). A value of 0 disables it.
 
-  You have a lot of flexibility on which voice channels you choose to play your melodies on.
-  For instance, you may decide to use only voice 1 and voice 4 for your melody, and spare
-  the other channels for melody-based sound effects (for simple one-shot sound effects,
-  consider the {\bf SOUND} command instead). Just skip the voices you're not using with
-  {\bf PLAY}, by leaving those arguments empty:
+  Similarly, the portamento directive slides between consecutive
+  notes at the speed you specify (1 -- 9). A value of 0 disables it. Note that
+  the gate-off behaviour of notes is disabled while portamento is enabled.
+  To re-enable the gate-off behavior, you must disable portamento (P0).
+
+  If a string ends with the {\bf L} directive, the pattern loops back to the
+  beginning of the string upon completion.
+
+  You can omit a string for a given voice, such as to allow a looping pattern
+  on that voice to continue playing, using empty arguments:
```

Comment:
Feel like the description now is a little too sparse to convey the intent of the capability.

Reply:
I see what you mean. This is a mild rewrite but I think the emphasis is better: "You can omit a string for a given voice to allow an already playing pattern in that voice to continue, using empty arguments:"